bin/persistent-setup

```diff
@@ -115,8 +122,18 @@ fi
 
     # Display memorization tips
     zenity --info --title='Tips for your new passphrase' \
-        --text="You will be prompted periodically to enter your passphrase as spaced repetition training. Practice entering your passphrase <i>daily at first and then at least once a week</i>  to commit it to memory.\n\nIt is essential to keep your passphrase safe. Do not re-use it for any other accounts. If you wrote it down, store it out of sight, in a secured location, away from your Bails USB sticks and destroy it once you memorize it." \
-        --ok-label='I Will Destroy It Once Memorized' --width=640 $ICON
+            --text="You will be prompted to enter your passphrase as spaced repetition training.\n\nPractice entering your passphrase <i>daily at first and then at least once a week</i>  to commit it to memory.\n
```

Comment:
@epiccurious: review of this dialog text would be nice. I went Autistic on it.  You can copy the code into Ubuntu terminal and it will mostly look the same as it would have in Tails if you don't have a VM open.